Rakefile

```diff
@@ -32,7 +32,7 @@ exclude_paths = [
 
 # Coverage from puppetlabs-spec-helper requires rcov which
 # doesn't work in anything since 1.8.7
-Rake::Task[:coverage].clear
+#Rake::Task[:coverage].clear
```

Comment:
Either delete the line or leave it in, but leaving the comment isn't a good idea.